build/js/live-editor.output_pjs.js

```diff
@@ -1602,10 +1614,38 @@ window.PJSOutput = Backbone.View.extend({
             // but PImage may be mutable, so that might not work.
             this.imageCache[file] = img;
         }.bind(this));
-    },
+        
+        _.each(sounds, function(file) {
```

Comment:
Hm, interesting. I thought about doing this but worried about a program with a lot of sounds suffering from a long initial load. But I suppose its better for it to suffer in the beginning than later. We might want to have a more helpful loading indicator in future to specify that it's loading images and sounds, I believe SCRATCH does that, since their programs are so media-rich.